scripts/main_script.sh

```diff
@@ -6,7 +6,7 @@ set -e
 if [ "$COMPONENT" == "BUILD" ]; then
     ./gradlew build
 elif [ "$COMPONENT" == "CONNECTED_TEST" ]; then
-    ./gradlew build connectedAndroidTest --stacktrace
+    ./gradlew build connectedAndroidTest jacocoTestReport --stacktrace
```

Comment:
Remove `jacocoTestReport` from here too.

Reply:
@abishekvashok Wanted it to be there.

Reply:
jacocoTestReports helps to collect unit test code coverages quickly

Reply:
@harshit98 I guess he should put it here as test coverage should be reported by jacoco for codecov for smooth working.

Reply:
ok great!